saos-webapp/src/main/resources/message/common.properties

```diff
@@ -110,16 +109,23 @@ judgment.judgmentResult = Skrócony wynik sprawy
 judgment.lowerCourtJudgments = Wyroki sądu niższej instancji
 
 /* Labels for landing page aka home */
-home.welcome.header = Wczesna wersja testowa Systemu Analiz Orzeczeń Sądowych!
-home.welcome.text = Sed ut perspiciatis unde omnis iste natus..
+home.beta.message = Serwis znajduje się obecnie w fazie <b>testów beta</b>. Uwagi i sugestie dotyczące działania serwisu SAOS oraz pomysły funkcji, które mogłyby się w nim znaleźć, uprzejmie prosimy przesyłać na adres
+home.helpdeskAddress = helpdesk@saos.icm.edu.pl
```

Comment:
It's not very good idea to store email address in messages file.
I think that more suitable place for this is in `saos.default.properties` file.